content/blog/2021/05/new-camel-k-logging-features/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "New Camel K Logging Features"
+date: 2021-05-10
+authors: ["orpiske"]
+categories: ["Features", "Camel K"]
+preview: "New Camel K Logging Features"
+summary: "New Camel K Logging Features for Camel K 1.5"
+---
+
+
+As part of our continuous work to improve Camel K and improve its stability, features and
+usefulness to our community, we recently worked on logging improvements for different
+parts of Camel K and the integrations it generated.
+
+During the 1.5 development release we dedicated some time to review the logging capabilities
+of Camel K. This version introduces a new [logging trait](https://camel.apache.org/camel-k/latest/traits/logging.html) that simplifies access to the logging configuration available on the runtime.
```

Comment:
```suggestion
of Camel K. This version introduces a new [logging trait](/camel-k/latest/traits/logging.html) that simplifies access to the logging configuration available on the runtime.
```

We need to go with relative links, the reason is that we don't check remote links and if we leave links to camel.apache.org as remote this link won't get checked and we would not know it's broken.

Reply:
Thanks Zoran! Let's do this.